jwst/fits_generator/main.py

```diff
@@ -28,7 +28,7 @@
 # DAMAGE.
 
 # THIRD-PARTY
-from astropy.io import fits as pyfits
+# from astropy.io import fits as pyfits
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/spacetelescope/jwst/pull/8630/commits/aae54e248caeab4aa099e93b9dd31e318ee76d1f